### PR TITLE
Fix some inconsistencies on dev tooling

### PR DESCRIPTION
**Pull request by rasjani**
* Add flake8 to dev dependencies
* Add possibility to run unittests via inv
* include tasks.py in lint target
* fixed some typos

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -208,7 +208,20 @@ def atest(ctx, suite=None):
...
+@task
+def utest(ctx, suite=None):
+    """Runs utest/run.py
+
+    Args:
+        suite: Select which suite to run.
+    """
+    command = "python utest/run.py"
+    if suite:
+        command = f"{command} --suite {suite}"
```
Comment (rasjani) [inline tasks.py:226]: doesnt seem to work ..

Comment (aaltat) [reply tasks.py:226]: That should work, at least last week it did work

Comment (rasjani) [reply tasks.py:226]: Could be pebcak error. But I was running --suite locators and console log was reporting plenty of other tests. Maybe they where skipped but still shown in terminal as I didn't pay attention to status.

Comment (rasjani) [reply tasks.py:226]: yeah, it was error between chair and monitor :D

**Pull request merged**